### DISCUSSION TITLE
How i can get shop receipts by ids

### DISCUSSION THREAD

**Discussion by imustafayasin** [etsy/open-api#874] [Q&A]
I want to get specific receipts by ids(int array) but there is no path parameter like this. Currently i can get single receipt. This feature was in v2.

v2: https://www.etsy.com/developers/documentation/reference/receipt#method_getshop_receipt2
v3 : https://developers.etsy.com/documentation/reference#operation/getShopReceipt

Could you advise me?

**Comment by Gareth064** [comment]
There is no endpoint to get a list of receipts by passing in the IDs. And Etsy have said in the past that they don't plan to introduce one #666
